src/Entity/QuizInstance.php

```diff
@@ -96,11 +96,11 @@ public function setName(string $name)
     }
 
     /**
-     * @param string $type
+     * @param string $description
      */
-    public function setType(string $type)
+    public function setDescription(string $description)
```

Comment:
return type

Reply:
Resolved in one of the 'search' PR you merged today.